ChocChipCookies.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Chocolate Chip Cookies Recipe</title>
+  </head>
+  <body>
+    <h1>Chocolate Chip Cookies</h1>
+    <br>Recipe from:<a href="https://www.kingarthurbaking.com/recipes/chocolate-chip-cookies-recipe">King Arthur Baking</a>
+    <br><img src="chocolatechipcookies.jpg" alt="Yummy Cookies" />
+    <h4>Preparation Time:</h4>36 mins
+    <p>
```

Comment:
For grouping h tags and lists together, I would recommend using a div rather since this is not a paragraph